docs/user-guide/deploy-chart/deployment-of-charts.md

```diff
@@ -29,45 +29,44 @@ Enter the following details, to deploy chart:
 | `Chart Version` | Chart version |
 | `Chart Value` | Latest default value or create a custom value |
 
-![](../../.gitbook/assets/charts-3.png)
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploy-chart/deployment-of-charts/charts-3.jpg)
 
 you can choose any chart version, values and update it on values.yaml
 
 Click on `Deploy` to deploy the Chart
 
 ## 3. Status of Deployment 
 
-After clicking on `Deploy` you will land on a page, that shows the Status of the deployment of the Chart.
-
-The Status of the chart should be `Healthy`. It might take few seconds after initiating the deployment of the chart.
+After clicking on `Deploy` you will land on a page that shows the status of the deployment of the Chart.
 
-![](../../.gitbook/assets/charts-4.png)
+The status of the chart should be `Healthy`. It might take few seconds after initiating the deployment of the chart.
 
-In case the Status, of the deployment is `Degraded` or takes a long time to get deployed.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploy-chart/deployment-of-charts/charts-4.jpg)
 
-Click on the `Status` or check the logs of the pods to debug the issue.
+In case the status of the deployment is `Degraded` or takes a long time to get deployed, click on `Details` in `Application Status` section on same page or check the logs of the pods to debug the issue.
 
 Arrow 3 will land you to the page where you can update or upgrade chart
```

Comment:
Please change this, `Arrow 3/4 will land.. ` is wrong. And where are 1 and 2?
Example:
3. Clicking on Configure will land you to the page where you can update, upgrade or delete chart.